pydantic_odm/db.py

```diff
@@ -46,14 +99,17 @@ def __class_getitem__(cls, item):
         cls.settings = settings
 
         for alias, configuration in cls.settings.items():
-            client = motor_asyncio.AsyncIOMotorClient(
-                username=configuration.get('USERNAME'),
-                password=configuration.get('PASSWORD'),
-                host=configuration.get('HOST', 'localhost'),
-                port=configuration.get('PORT', 27017),
-                authSource=configuration.get('AUTH_SOURCE', 'admin'),
-                authMechanism=configuration.get('AUTH_MEC', 'SCRAM-SHA-256'),
-            )
+            connection_params = {
```

Comment:
It seems to me that this configuration option will be simpler and more understandable. 
If something goes wrong - the driver (motor, pymongo) itself will tell us about it directly. 
In our case, this give is a minimum of logic for connecting to the MongoDB.